apps/nrm/behaviours/confirmation.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+/**
+ * Fomrat heading text based on user journey ( NRM or Duty to Notify)
+ *
+ * @param {Object} req - request object
+ *
+ * @returns {Object} - formatted data for display on UI
+ */
+const formatHeading = (req) => {
+  let data = {};
+  let headingMessage = 'NRM referral sent';
+
+  if (req.sessionModel.get('pv-want-to-submit-nrm') === 'no') {
+    headingMessage = 'Duty to Notify (DtN) sent';
+  }
+
+  data = Object.assign({}, data, {
+    confirmationHeading: headingMessage,
+  });
+
+  return data;
+};
+
+module.exports = superclass => class extends superclass {
+  locals(req, res) {
+    const superlocals = super.locals(req, res);
+    const data = Object.assign({}, formatHeading(req));
+    const locals = Object.assign({}, superlocals, data);
+
+    return locals;
+  }
+};
```

Comment:
I’d probably rewrite this code

Can this all be simplified like this
https://github.com/UKHomeOffice/modern-slavery/blob/master/apps/verify/behaviours/confirm-email-passer.js